### PR TITLE
feat: Update document

### PR DESCRIPTION
**Pull request by JeffreyDallas**
## Description

This pull request changes the following:

* Add section for using environment variables and FAQ
* Update Dev.md

### Related Issues

#914

### CODE REVIEW

File: Taskfile.yml [hunk 1]
```diff
@@ -58,7 +58,6 @@ tasks:
   solo:mirror-node:
     desc: solo mirror-node deploy with port forward on explorer
     cmds:
-      - npm run build
```
Comment (jeromy-cannon) [inline Taskfile.yml:61]: We need to make a task that will run build and leave an artifact behind somewhere, perhaps in temp directory with the filename as a timestamp set at startup.  That way, we can run the build one time, but check before any call.  If you remove the npm run build, then we run the risk that they may make a call that runs on an older build of code and does not match the code they have checked out on their machine.

File: Taskfile.yml [hunk 2]
```diff
@@ -70,7 +69,6 @@ tasks:
     status:
       - helm list -n "${SOLO_NAMESPACE}" | grep -vqz "${MIRROR_RELEASE_NAME}"
     cmds:
-      - npm run build
```
[the remark this answers is not in the document]
Comment (JeffreyDallas) [reply Taskfile.yml:73]: How about user to run `npm run build` before using Task

https://github.com/hashgraph/solo/blob/main/docs/content/User/TaskTool.md

[the remark this answers is not in the document]
Comment (jeromy-cannon) [reply Taskfile.yml:73]: see my changes, here: https://github.com/hashgraph/solo/pull/978

[the remark this answers is not in the document]
Comment (JeffreyDallas) [reply Taskfile.yml]: reverted the change

File: Taskfile.yml (hunk 1 above)
Comment (JeffreyDallas) [reply Taskfile.yml]: reverted the change

**Review by jeromy-cannon — APPROVED** [review]

**Pull request merged**